### PR TITLE
docker action first test

### PR DESCRIPTION
**Pull request by IAmMorrow**
This PR add a spectron test using a docker container on the CI github action.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -35,3 +35,6 @@ jobs:
           yarn prepublish
       - name: test
         run: yarn ci
+      - name: Run spectron
+        id: spectron
+        uses: IAmMorrow/run-headless@v4
```
Comment (gre) [inline .github/workflows/ci.yml:40]: (can we later move it to LedgerHQ? would need maybe a more obvious name, like `ledger-live-github-action-headless` or something)

Comment (valpinkman) [reply .github/workflows/ci.yml:40]: If it works, you can move it 
https://github.com/LedgerHQ/actions

Comment (IAmMorrow) [reply .github/workflows/ci.yml:40]: yep and we'll take the command to run as input

**Comment by gre** [comment]
ok let's merge this for now. :goodenough:
and we can iterate in another PR for the small changes of actions

**Pull request merged**